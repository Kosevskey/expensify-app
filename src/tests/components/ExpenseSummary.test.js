import React from "react";
import { ExpenseSummary } from "../../components/ExpenseSummary";
import { shallow } from 'enzyme';

test('should correctly render ExpenseSummary with 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpenseSummary with multiple expenses', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={23} expensesTotal={2350987456} />);
    expect(wrapper).toMatchSnapshot();
});


