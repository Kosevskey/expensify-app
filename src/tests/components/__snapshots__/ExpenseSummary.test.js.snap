// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpenseSummary with 1 expense 1`] = `
<div>
  <p>
    Viewing 
    235
     
    expense
     totalling 
    $2.35
    .
  </p>
</div>
`;

exports[`should correctly render ExpenseSummary with multiple expenses 1`] = `
<div>
  <p>
    Viewing 
    2350987456
     
    expenses
     totalling 
    $23,509,874.56
    .
  </p>
</div>
`;
